client/src/components/startPage.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
+import Header from "./header";
+
+class StartPage extends Component {
+    constructor(props) {
+        super(props);
+
+        this.getPeopleInQueue();
+    }
+    state = {
+        queue: 0,
+    };
+    //This method asks the server for the size of the queue of people using POST and JSON.
+    getPeopleInQueue = () => {
+        const requestOptions = {
+            method: "POST",
+            headers: { "Content-Type": "application/json" },
+        };
+        fetch("/api/queue", requestOptions).then((response) => {
+            response.json().then((body) => {
+                this.setState({ queue: body.queueSize });
+            });
+        });
+    };
+
+    render() {
+        return (
+            <div className="text-center border-bottom-3">
+                <Header />
+                <h1> Välkommen till flippen!</h1>
+                <h4>Det är just nu {this.state.queue} pers i kö </h4>
+                <button className="btn btn-lg  btn-primary m-4">
+                    Till beställning
+                </button>
+                <button className="btn btn-lg  btn-info m-4">Se kö</button>

```

Comment:
Ser lite loj ut.
Tror dessa changes hade gett en lite mer "seriös" känsla.
```suggestion
                <h1> Välkommen till Flippen!</h1>
                <h4>Det är just nu {this.state.queue} personer i kön </h4>
                <button className="btn btn-lg  btn-primary m-4">
                    Till beställning
                </button>
                <button className="btn btn-lg  btn-info m-4">Se kön</button>
```